### ISSUE TITLE
Genes driving the prediction

### ISSUE THREAD

**Issue by cdiaz45** [Teichlab/celltypist#45]
Hello, as usual thank you so much for all your help with my questions. I am wondering if there is a way to know which genes are driving a specific similarity with a model? Can I get that information from the results?
Many thanks,
Carmen

**Comment by ChuanXu1** [comment]
@cdiaz45, as indicated in the [tutorial](https://celltypist.readthedocs.io/en/latest/notebook/celltypist_tutorial.html#Examine-expression-of-cell-type-driving-genes), the driving genes of each cell type is solely based on training dataset / model. You can obtain the markers using the `extract_top_markers` function. Is this what you are asking?

**Comment by cdiaz45** [comment]
Ah no problem. I actually built the model using all of the genes so I wanted to see which ones contributed the most. A friend suggested to look at the beta coefficient (although I am repeating his words), so I wanted to see if it was possible to extract those. I saw you have several versions on how to build the model so will try different ones and check how much that changes the predictions.  Thank you though. Amazing work and tool.
